data/src/main/kotlin/info/dvkr/screenstream/data/httpserver/HttpServerRxHandler.kt

```diff
@@ -106,8 +103,9 @@ internal class HttpServerRxHandler(
             uri == HttpServerFiles.FULLSCREEN_ON_PNG_ADDRESS -> response.sendPng(httpServerFiles.fullScreenOnPng)
             uri == HttpServerFiles.FULLSCREEN_OFF_PNG_ADDRESS -> response.sendPng(httpServerFiles.fullScreenOffPng)
             uri.startsWith(systemActionAddress) -> onSystemAction(uri.substringAfter("?")).run {response.empty() } // possible get rid of sending a response...
-            uri == HttpServerFiles.DEFAULT_HTML_ADDRESS -> response.sendHtml(if (pinEnabled) pinRequestHtml else indexHtml)
-            uri == pinAddress && pinEnabled -> response.sendHtml(indexHtml)
+            uri == HttpServerFiles.DEFAULT_HTML_ADDRESS -> response.sendHtml(if (pinEnabled) pinRequestHtml else
+                httpServerFiles.configureDirectorHTML(streamAddress, appActionAddress, systemActionAddress))
+            uri == pinAddress && pinEnabled -> response.sendHtml(httpServerFiles.configureDirectorHTML(streamAddress, appActionAddress, systemActionAddress))
```

Comment:
Great idea. I need to go in and configure the webpage to trigger a refresh when settings like picture quality are changed. Right now it just kills the webpage and you have to refresh it manually from the browser.